### DISCUSSION TITLE
osu!mania Hold Note End seems to be shorter and stranger than stable with custom skin

### DISCUSSION THREAD

**Discussion by Nana-Sakura** [ppy/osu#25991] [Q&A] [answered]
![osu_2023-12-21_12-07-48](https://github.com/ppy/osu/assets/60061683/1b2e442a-bc87-4cb6-8618-331b84001f08)
![screenshot625](https://github.com/ppy/osu/assets/60061683/c98624db-51a0-4650-bba2-a255f38bc52c)
First picture is in Lazer, which Hold Note End is shorter than the next one in Stable, and the shape seems to be stranger too.
So how to fix the skin or is it a client issue?

**Comment by peppy** [comment, marked as answer]
The issue is that your skin has 30,000 pixel elements, which we don't support properly (it's-not-easy)

**Reply by Nana-Sakura** [reply]
OK

**Reply by Nana-Sakura** [reply]
Tried to expand the element mania-noteL.png to 40,000 pixels and fixed the issue. Original one is only 20,000 pixels and had caused some unexpected behaviors, which is also fixed. Thank you again for your answer.
